### PR TITLE
IGAPP-882: LanguageNotAvailablePage not scrollable

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/components/Selector.tsx [hunk 1]
```diff
@@ -1,98 +1,30 @@
...
+import React, { ReactElement } from 'react'
 import styled from 'styled-components/native'
 
-import { ThemeType } from 'build-configs'
-
-import dimensions from '../constants/dimensions'
 import SelectorItemModel from '../models/SelectorItemModel'
+import SelectorItem from './SelectorItem'
 
-const ItemWrapper = styled.View<{ selected: boolean }>`
```
Comment (klinzo) [inline native/src/components/Selector.tsx:12]: Extracted to `Selector` component

File: native/src/components/Selector.tsx [hunk 2]
```diff
@@ -1,98 +1,30 @@
...
   selectedItemCode: string | null
-  theme: ThemeType
 }
 
-/**
- * Displays a Selector showing different items
- */
-
-class Selector extends React.Component<PropsType> {
-  getItems(): React.ReactNode {
```
Comment (klinzo) [inline native/src/components/Selector.tsx:61]: Extracted to separate `Selector` component

File: native/src/routes/ChangeLanguageModal.tsx [hunk 3]
```diff
@@ -13,56 +11,46 @@ import SelectorItemModel from '../models/SelectorItemModel'
...
+  const selectorItems = languages.map(({ code, name }) => {
+    const isLanguageAvailable = availableLanguages.includes(code)
+    return new SelectorItemModel({
+      code,
+      name,
+      enabled: isLanguageAvailable,
+      onPress: () => {
+        if (code !== currentLanguage) {
+          changeLanguage(code)
+        }
```
Comment (klinzo) [inline native/src/routes/ChangeLanguageModal.tsx:43]: This is new. Also removed `InteractionManager` as the following is definitely not true for a simple dispatch:
> InteractionManager allows long-running work to be scheduled after any interactions/animations have completed. In particular, this allows JavaScript animations to run smoothly.

File: native/src/components/LanguageNotAvailablePage.tsx [hunk 4]
```diff
@@ -1,61 +1,42 @@
...
 import Caption from './Caption'
 import Selector from './Selector'
 
-const Wrapper = styled.View`
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
+const Wrapper = styled.ScrollView`
```
Comment (klinzo) [inline native/src/components/LanguageNotAvailablePage.tsx:12]: The important change in this PR

**Review by f1sh1918 — APPROVED** [review]
tested on ios. nice refactoring :)

File: native/src/components/__tests__/Feedback.spec.tsx [hunk 5]
```diff
@@ -1,7 +1,8 @@
-import { fireEvent, render } from '@testing-library/react-native'
+import { fireEvent } from '@testing-library/react-native'
 import React from 'react'
 
 import buildConfig from '../../constants/buildConfig'
+import render from '../../testing/render'
```
Comment (Taggotty) [inline native/src/components/__tests__/Feedback.spec.tsx:5]: What's this about?

**Review by Taggotty — APPROVED** [review]
Well done. Also the tests 🔥 
Tested on Android emulator

File: native/src/components/__tests__/Feedback.spec.tsx (hunk 5 above)
Comment (klinzo) [reply native/src/components/__tests__/Feedback.spec.tsx:5]: Wraps the test automatically with `wrapWithTheme`. In web this does even more (like wrapping in a router), but for native this is all it does at the moment. So just a little helper.

**Pull request merged**